catroid/src/main/res/values-ar/strings.xml

```diff
@@ -258,18 +316,18 @@
   <string name="dialog_terms_of_use_parrot_reminder_do_not_remind_again">Do not remind me again.</string>
   <!-- About Dialog -->
   <string name="dialog_about_title">حول تطبيق الرسام</string>
-  <string name="dialog_about_license_info">Pocket Code is a programming 
-        environment and interpreter for Android devices for the visual 
+  <string name="dialog_about_license_info">Pocket Code is a programming
+        environment and interpreter for Android devices for the visual
         programming language Catrobat.\n\n
-        The source code of Pocket Code is mainly licensed under the GNU AGPL v3 License. 
+        The source code of Pocket Code is mainly licensed under the GNU AGPL v3 License.
         For further details on the license see the link below.\n</string>
   <string name="dialog_about_pocketcode_license_link_text">رخصة مصدر النص البرمجي لجهاز الجيب</string>
   <string name="dialog_about_catrobat_link_text">حول كاتروبات</string>
   <!-- Project activity view -->
   <string name="project_name">البرنامج:</string>
   <string name="my_projects_activity_title">البرامج</string>
   <string name="number_of_objects">Objects:</string>
-  <string name="start_scene_name">\"%1$s\" (First Scene)</string>
+  <string name="start_scene_name">"%1$s" (First Scene)</string>
```

Comment:
I'm not sure if this is on purpose. Quotes that aren't escaped, will be stripped.
https://jira.catrob.at/browse/CAT-2147